snuba/datasets/discover.py

```diff
@@ -55,7 +55,7 @@
 logger = logging.getLogger(__name__)
 
 
-def detect_table(
+def detect_dataset(
```

Comment:
Why was this renamed to detect_dataset ? We are dealing with one dataset here and multiple storages, the name would be misleading.

Reply:
Reverted this